libs/scraper-archive/src/overrides.ts

```
@@ -0,0 +1,35 @@
+  // If elements is specified, only skip those elements
+  elements?: string[],
+}
+export type SkipElements = Record<string, SkipElement>
+export type OverrideData = {
+  skip?: SkipElements,
+  overrides?: Record<string, Record<string, OverrideElement>>
+}
+
+export function getOverrideData(): OverrideData {
+  const overrideFile = tests("archive", "overrides.json")
+  if (existsSync(overrideFile)) {
```

ESLint

[error] Missing semicolon.

(semi)